sigmadb.py

```diff
@@ -91,7 +91,7 @@ def main():
 
                 action = ""
                 while action != "1":
-                    action = input_utils.get_input_matching("1 - exit\n2 - browse movies\n> ", regex="[12]")
+                    action = input_utils.get_input_matching("1 - exit\n2 - browse movies\n3 - following\n", regex="[123]")
```

Comment:
The submenu name "following" doesn't make too much grammatical sense w/ the prompt imo. Perhaps "manage followed users" instead?